### PR TITLE
Cleaned up, added with documentation on config info.

### PR DESCRIPTION
**Pull request by pron3211**
i think it works (key word think)

### CODE REVIEW

File: SConstruct [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 import os
 
-GNU_PATH = SConscript('config.txt')
```
Comment (BaylorWhitehead) [inline SConstruct:5]: Revert this

File: template/generic/board/can.h [hunk 2]
```diff
@@ -0,0 +1,122 @@
+/*
```
Comment (BaylorWhitehead) [inline template/generic/board/can.h:1]: Remove this whole file

File: template/generic/source/main2.cpp [hunk 3]
```diff
@@ -0,0 +1,12 @@
+#include "MKE18F16.h"
```
Comment (BaylorWhitehead) [inline template/generic/source/main2.cpp:1]: Remove this whole file

**Review by BaylorWhitehead — CHANGES_REQUESTED** [review]
Also remove the .eld and libbsp.a file. Also look at other comments

**Comment by pron3211** [comment]
good?

**Review by BaylorWhitehead — APPROVED** [review]

**Pull request merged**